app/jobs/sms_sender_number_change_job.rb

```diff
@@ -18,7 +19,7 @@ def number_change_message
     I18n.t(
       'jobs.sms_sender_number_change_job.message',
       app: APP_NAME,
-      support_url: Figaro.env.support_url
+      support_url: contact_url
```

Comment:
I am not sure...the URL was not shortened before, so if we need to shorten it would not be a new problem